src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java

```diff
@@ -25,14 +28,22 @@
 @Service
 public class FilingsService {
 
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d MMMM yyyy");
+
+    @Value("${OVERSEAS_ENTITIES_FILING_DESCRIPTION_IDENTIFIER}")
```

Comment:
Hi, do we need to add the following `OVERSEAS_ENTITIES_FILING_DESCRIPTION_IDENTIFIER` to our configs?

Reply:
Yes. PRs to follow to do this (in Docker and chs-configs). Feel free to approve this PR but I won't merge until all PRs are ready to go.